trackintel/geogr/distances.py

```diff
@@ -212,21 +212,22 @@ def check_gdf_crs(gdf, transform=False):
     >>> check_gdf_crs(triplegs, transform=False)
     """
     if_planer = False
-    if gdf.crs is None:
-        # projection is not defined
+    if gdf.crs is None:  # projection is not defined
         if_planer = False
         if transform:
             gdf.crs = "EPSG:4326"
         else:
             warnings.warn("Your data is not projected.")
 
-    elif gdf.crs == "EPSG:4326":
-        # if projection is defined as WGS84
+    elif gdf.crs == "EPSG:4326":  # if projection is defined as WGS84
         if_planer = False
 
-    else:
-        # if projection is defined but not as WGS84
-        if_planer = True
+    else:  # if projection is defined but not as WGS84
+        if gdf.crs.is_geographic:  # if projection is a geographic crs
+            if_planer = False
+        else:
+            if_planer = True
+
         if transform:
```

Comment:
I would change the indentation. At the moment it would not be transformed if gdf.crs is None but transform is true.

Reply:
I think transform is only valid in the else statement (# if projection is defined but not as WGS84). 

If gdf.crs is None but transform is true, the function executes line 218, where the crs is defined as 84, here transform is an alias of define/set